modxcms/revolution
# Fix Element handling from within Categories tree

### What does it do?
Adds a new tree method to properly extract Element data from the working tree Node. There are three patterns for a Node's id, but only one was being accounted for in the code. Additionally, cleans up code formatting in the affected methods.

### Why is it needed?
Within the Categories tree, these actions are currently inoperable:
- Element removal
- Plugin activation/deactivation

### How to test
1. Create a handful of Elements (at least one of each type), both within a Category and within the root (no Category)
2. Verify the ability to remove each Element (from both the Categories tree and from within the individual Element trees)
3. Verify the ability to activate/deactivate Plugins (from both the Categories tree and from within the individual Element trees)

### Related issue(s)/PR(s)
Resolves #16487 for MODX 3.x
Related to 2.x solution #16488
